front/src/applications/editor/components/EntityError.tsx

```diff
@@ -26,19 +26,21 @@ const EntityError: FC<{ entity: EditorEntity; className?: string }> = ({ entity,
     return data?.results && data.results.length > 0;
   }, [infraID, data]);
 
-  return hasError ? (
-    <div className={cx('entity-errors-linked', className)}>
-      <h4>
-        <BsExclamationOctagon className="mr-1" />
-        {t('Editor.entities.errors-linked')}
-      </h4>
-      <div className="small">
-        {data?.results?.map((e) => (
-          <InfraErrorLine key={uniqueId()} error={e.information} />
-        ))}
+  return (
+    hasError && (
```

Comment:
not a big fan of the `&&` operator at the top of the JSX, it’s better than the ternary, but in a react component we expect to see a component being returned, here the function is returning an && expression.

I believe the better approach is to return null early, which is what they propose in 
[react doc](https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null)

Reply:
Relevant remark thanks, I change that :)